src/types.md

```diff
@@ -428,13 +499,68 @@ Box<Bar<'a>>
 Box<Bar<'a> + 'a>
 ```
 
-The default for user-defined trait objects is based on the object type itself.
-If a type parameter has a lifetime bound, then that lifetime bound becomes the
-default bound for trait objects of that type. For example, `std::cell::Ref<'a,
-T>` contains a `T: 'a` bound, therefore trait objects of type `Ref<'a,
-SomeTrait>` are the same as `Ref<'a, (SomeTrait + 'a)>`.
-
-### Type parameters
+The default for user-defined types with a trait object used as a type parameter
+is based on the type itself. If a type parameter has a lifetime bound, then
+that lifetime bound becomes the default bound for trait objects of that type.
+For example, `std::cell::Ref<'a, T>` contains a `T: 'a` bound, therefore trait
+objects of type `Ref<'a, SomeTrait>` are the same as `Ref<'a, (SomeTrait +
+'a)>`. If there's no such bound then the default bound is `'static`, for
+example in `std::rc::Rc<T>`.
+
+
+## Dynamically sized types
```

Comment:
The more I look at this section, the more I wonder if this section should be on the Sized trait itself. The other types in this chapter (except `Self`...) are non-overlapping, whereas DSTs are a facet of a type.